tests/fixtures/rules/require-spread/bad/example.js

```diff
@@ -0,0 +1,2 @@
+const wrap = (f, g) => (...args) => g.apply(g, [f].concat(args));
+instance.method.apply(instance, args);
```

Comment:
Can you move this second example into a separate file? If we don't do that the various types of bad things are guaranteed to fail...